extensions/wikia/Search/classes/Services/CombinedSearchService.php

```diff
@@ -180,22 +226,29 @@ protected function queryPhraseSolrForArticles( $query, $namespaces, $lang, $hubs
 
 		$select->createFilterQuery( 'dis' )->setQuery( '-(title_en:disambiguation)' );
 		//speedydeletion: 547090, scratchpad: 95, lyrics:43339,
-		$select->createFilterQuery( 'banned' )->setQuery( '-(wid:547090) AND -(wid:95) AND -(wid:43339) AND -(host:*answers.wikia.com)' );
+		$select->createFilterQuery( 'banned' )->setQuery(
+			'-(wid:547090) AND -(wid:95) AND -(wid:43339) AND -(host:*answers.wikia.com)'
+		);
 		if ( $minArticleQuality != null ) {
-			$select->createFilterQuery( 'article_quality' )->setQuery( 'article_quality_i:[' . $minArticleQuality . ' TO *]' );
+			$select->createFilterQuery( 'article_quality' )->setQuery(
+				'article_quality_i:[' . $minArticleQuality . ' TO *]'
+			);
 		}
 
 		$boostedTypes = [ "character", "tv_series", "tv_episode", "person", "move", "video_game" ];
-		$dismax->setBoostQuery( '+(wikititle_en:"' . $phrase . '"^100000)' . " +(article_type_s:(" . implode( " OR ", $boostedTypes ) . "))" );
+		$dismax->setBoostQuery(
+			'+(wikititle_en:"' . $phrase . '"^100000)' . " +(article_type_s:(" . implode( " OR ", $boostedTypes ) . "))"
+		);
 
 		$result = $client->select( $select );
+
 		return $this->extractData( $result, $requestedFields );
 	}
 
 	public function queryPhraseForWikias( $query, $hubs, $lang ) {
 		$fields = [ "url", "lang_s", "sitename_txt", "id", "score", "description_txt" ];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "sitename_txt" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "lang_s" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "description_txt" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)